metadata_tools.py

```diff
@@ -71,7 +74,7 @@ def exif_attach_metadata(self, exif_code: int, exif_value):
             exif_dict: dictionary of exif terms using exif codes, and new values to assign"""
 
         exif_tag = self.exif_code_to_tag(exif_code)
-        command = ['exiftool', f"-{exif_tag}={exif_value}", self.path]
+        command = ['exiftool', '-overwrite_original', f"-{exif_tag}={exif_value}", self.path]
```

Comment:
added command overwrite original to prevent  _original backup copies from being created.